docs/SNMP_Alarms.md

```diff
@@ -41,3 +41,13 @@ obtain the brief description, detailed description, and severity for the alarm:
 ## Alarm Models
 
 The alarm models used by Clearwater are defined in [alarmdefinition.h](https://github.com/Metaswitch/cpp-common/blob/master/include/alarmdefinition.h).
+
+## Alarm Resiliency
+
+Alarms are reported in such a way that if a component receiving alarm
+notifications were to fail, when it recovers it would regain knowledge of the 
+current alarm state of the system. This is partly because Clearwater nodes 
+keep track of the errors they have reported and re-report these every thirty 
```

Comment:
"re-report these every thirty seconds"

I think this is potentially confusing for the reader because it's mixing a couple of different levels of the code. On an **internal interface** - over ZMQ between a process and the alarms agent - we re-report alarms every thirty seconds. I don't think we do, or should, re-report all alarms over SNMP every thirty seconds - but your phrasing might make a reader think that, especially since this document is called "SNMP Alarms".